### PR TITLE
chore: adds android deploy script

### PR DESCRIPTION
**Pull request by redreceipt**

### CODE REVIEW

File: scripts/deploy-ios.sh [hunk 1]
```diff
@@ -1,15 +1,15 @@
 TRACK=$2
 
 if [ -z "$CHURCH" ] || [ -z "$APOLLOS_API_KEY" ]; then
-    echo "CHURCH and APOLLOS_API_KEY required!"
-    exit 1
+  echo "CHURCH and APOLLOS_API_KEY required!"
+  exit 1
 fi
 
-# if [ -n "$(git status --porcelain)" ]; then
```
Comment (vinnyjth) [inline scripts/deploy-ios.sh:8]: sorry about that

**Comment by vinnyjth** [comment]
any way I can help with this?

**Comment by redreceipt** [comment]
I think it's ready to go, I just wanted to test locally on the demo app before giving it a thumbs up. and we're waiting on a new play store account to be able to test it.

**Comment by vinnyjth** [comment]
https://github.com/ApollosProject/apollos-apps/pull/2564/files#diff-cd854c492f61e2f6743d7a0ec2edc5dd27638c13b6bb5ddc8014ba8c3f3ed805R68-R81

**Review by vinnyjth — APPROVED** [review]

**Pull request merged**